oeps/processes/oep-0056-architectural-process.rst

```diff
@@ -0,0 +1,218 @@
+
+We believe that ADRs - and more importantly, the conversations that happen around them - can be so much more.  We can use these documents and conversations as a vehicle for decentralizing and scaling our decision making process, making more informed decisions, democratizing access to domain knowledge, elevating our architectural discourse, and recording our decisions for posterity with additional contextual information that's often missing today.
+
+Finding expertise is difficult
+------------------------------
+
+Related to the above, historically it is also very difficult to find experts and stakeholders in a given domain.  Generally this happens via word of mouth, sometimes not at all, or occasionally by engineers realizing a vacuum of expertise exists and *they are now the expert!*
+
+This difficulty is compounded by the decentralized and global nature of the Open edX community.
+
+Atrophy of architectural principles and tech radar
+--------------------------------------------------
+
+An important component of our ability to make architectural decisions that are aligned across teams and organizations is measuring and vetting those decisions against a set of `shared architectural principles <https://openedx.atlassian.net/wiki/spaces/AC/pages/921895082/Architecture+Vision+Principles>`_ and technologies in our `Tech Radar <https://github.com/openedx/openedx-tech-radar>`_.  Today, we lack organized maintenance of our principles and radar, nor do we have an appropriate forum in which to discuss and socialize them.
+
+Missing ceremonies
+------------------
+
+As a community of companies invested in the success of the Open edX platform, we lack any sort of regular ceremonies in which we discuss the platform's architectural priorities as they relate to our desired platform and organization-specific goals.  This often leads to a situation where architectural thinking is done primarily by senior engineering leaders as they learn about their business' needs, and by engineers in the course of projects, with little structured decision making in between to help bridge the highly strategic and the tactical.  Furthermore, the conversations that do happen tend to occur in an ad hoc way - often as "hallway conversations" or one-off project syncs - with whoever can be found to offer enough input to unblock the team.
+
+Specification
+=============
+
+We believe that we can create a healthy and empowering dialogue around our architectural process and mitigate the above concerns by focusing on a re-alignment of ceremonies and clarifying the spirit and process around the creation and review of ADRs.
+
+We will adjust our current architectural process by:
+
+- Formalizing and clarifying an **Architectural Advisory Process** in which we conversationally provide input on and promptly review the status of ADRs, as well as review updates to our architectural principles and tech radar, with the goal of enabling more effective, informed decision making.
```

Comment:
Nit: I'm not certain what you mean by `conversationally provide input`. Do you mean casually? Do you mean synchronously?

Reply:
I threw something in the principles about this.  Also, it's a term from the supporting articles linked in the motivation.

Reply:
OK. Please note I'm reading this with fresh eyes - I was not involved in any discussion and am reading this as a new person with no context would begin to read the OEP.  A lot of my suggestions are in the realm of clarity and approachability, as I found a number of things confusing. I think if you're expecting people to read references, you should footnote them in-line.